Alignment/CommonAlignmentProducer/src/AlignmentProducerBase.cc

```diff
@@ -854,7 +855,13 @@ void AlignmentProducerBase::writeForRunRange(cond::Time_t time) {
     auto alignments = alignableTracker_->alignments();
     auto alignmentErrors = alignableTracker_->alignmentErrors();
     this->writeDB(
-        alignments, "TrackerAlignmentRcd", alignmentErrors, "TrackerAlignmentErrorExtendedRcd", trackerGlobal, time);
+        // ~alignments, "TrackerAlignmentRcd", alignmentErrors, "TrackerAlignmentErrorExtendedRcd", trackerGlobal, time);
```

Comment:
Can this commented out and outdated line get removed?

Reply:
Addressed at https://github.com/cms-sw/cmssw/commit/1ac393ce9dbd37d6f1944233b100b527425611ea